examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        "data_acquisition", activity_function=acquire_data_async, power_consumption_in_watt=266
</pre><p>You need to set the constraint function here or the plot will not update during this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
This is on purpose. I added the last activity that lasts long to see the plot, while keeping the other activities with reasonable time without necessarily plotting their status. Also, from the didactic point of view, I think it's better to introduce the constraint later on.